ZenPacks/zenoss/OpenStackInfrastructure/apiclients/txapiclient.py

```diff
@@ -77,39 +77,43 @@ def __init__(self, username, password, auth_url, project_id, is_admin=False):
 
     @property
     def keystone_endpoints(self):
-        if '/v2' in self.auth_url:
+        if '/v2' in self.auth_url and self.is_admin is not None:
```

Comment:
I would use True/False rather than None / not None for this value.     Also might want to provide a cleaner way to test API version in use..

Also, i'd suggest moving the 'if' into self._admin_only, and just make it like "self._v2_admin_only" or something.  Less duplication that way.